doc/release-notes.md

```diff
@@ -4,3 +4,11 @@ release-notes at release time)
 Notable changes
 ===============
 
+
+Sprout to Sapling Migration Tool Fixes
+--------------------------------------
+This release includes fixes to the Sprout to Sapling Migration Tool found in
+testing. We resolved an issue which would cause the zcash daemon to crash if
+calling the RPC `z_getmigrationstatus` while a wallet's migration transactions
+are in the mempool. We have also added logic to prevent a race condition which
```

Comment:
I don't think this last sentence is needed, and as worded, is not clear what might have been fixed.